### DISCUSSION TITLE
Issue for the example

### DISCUSSION THREAD

**Discussion by hectorrrr6** [denisneuf/python-amazon-ad-api#199] [Announcements]
**Describe the bug**
When I use the example that Provided: 
```
from ad_api.api import sponsored_products
my_credentials = dict(
     refresh_token='************************',
      client_id='************************',
      client_secret='************************',
)
result = sponsored_products.Campaigns(credentials=my_credentials).list_campaigns()
print(result)
```
Get the error:
```
     self.credentials = CredentialProvider(account, credentials, verify_additional_credentials).credentials
AttributeError: 'CredentialProvider' object has no attribute 'credentials'. Did you mean: 'load_credentials'?
```

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by denisneuf** [comment]
my_credentials = dict(
    refresh_token='your-refresh_token',
    client_id='your-client_id',
    client_secret='your-client_secret',
    profile_id='your-profile_id',
)

**Comment by denisneuf** [comment]
profile_id='your-profile_id',
